src/pages/AddBook/addbook.js

```diff
@@ -0,0 +1,39 @@
+import { v4 as uuid } from 'uuid';
+import PropTypes from 'prop-types';
+
+const AddBook = ({ setList }) => {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const uniqueID = uuid();
+    const id = uniqueID.slice(0, 4);
+    const name = e.target.title.value;
+    const author = e.target.author.value;
+    const newList = {
```

Comment:
- [ ] It would be better to move the add book form component inside the Components folder directory. 
- Note: It's one of the React best practices if the pages directory could only contain page components: `<Books>` and `<Categories>` . Please address this 👍🏻